tgui/packages/tgui/interfaces/PocketDimensionCustomizer.js

```diff
@@ -0,0 +1,141 @@
+import { useBackend } from '../backend';
+import { Box, Button, Section, NumberInput } from '../components';
+import { Window } from '../layouts';
+
+export const PocketDimensionCustomizer = (props, context) => {
+  const { act, data } = useBackend(context);
+  return (
+    <Window
+      title="Pocket Dimension Customizer"
+      width={300}
+      height={500}
+      resizable>
+      <Window.Content>
+        <Box align="center" my={1}>
+          <Box >
+            Dimension Controls
+          </Box>
+
+          <Box>
+            <Button
+              content="Create Dimension"
+              onClick={() => act('create_dimension')}
+              tooltip={`DO THIS FIRST, causes some lag, may \
```

Comment:
1. With backticks, you do not need to escape newlines with backslashes.
2. Use the multiline string tag, because it removes all unnecessary whitespace from the string at compile time, and is better for the bundle size. For it to work, you must treat it like a separate block of text.

Example:
```js
multiline`
  Foo Bar
  Hippity Hoppity
`
```

How it's stored in the bundle:

```js
"Foo Bar/nHippity Hoppity"
```